python/tests/res/fm/test_fm_config.py

```diff
@@ -0,0 +1,39 @@
+import os
+import unittest
+from tests import ResTest
+
+class TestFMValidity(ResTest):
+
+    def setUp(self):
+        pass
+
+    def _extract_executable(self,filename):
+        executable_list = []
+        with open(filename,'r') as f:
+            for line in f.readlines():
+                l = [a.strip() for a in line.split(' ') if a not in ['', ' ', '  ']]
```

Comment:
This can be replaced by line.strip().split()

Reply:
I touhg about this in the first place, but was not sure about how this will handle irregular spaces, but sure can change it.

Reply:
It will handle tabs and what not. It is the default splitting in _Python_